cypress-io/cypress
# misc: Debug header updates

- Closes #25798 
- Closes #25796

### Additional details


This PR does two things:
- Updates the formatting of the run duration in the header from `HH:mm:ss` to `HH[h] mm[m] ss[s]` to avoid it looking like a time rather than a duration
- Fetches the commit author's Gravatar based on their email address (if there is no Gravatar, it falls back to the default icon that was there before)

### Steps to test


1. Run the `DebugPageHeader.cy.tsx` component test as well as `debug.cy.ts` e2e test and look at the output
2. View the Percy snapshot changes
3. View the debug page for an internal package and make sure that the duration formatting is correct, and a Gravatar is displayed for the commit author (if there is a commit author and if there is a Gravatar, otherwise look for the default icon)

### How has the user experience changed?


Check out the Percy build

### PR Tasks


- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
